curp/src/test_utils/test_cmd.rs

```diff
@@ -256,12 +263,8 @@ impl TestCESimple {
     }
 }
 
-pub(crate) fn random_id() -> ProposeId {
-    ProposeId::new(
-        thread_rng()
-            .sample_iter(&Alphanumeric)
-            .take(4)
-            .map(char::from)
-            .collect(),
-    )
+static NEXT_ID: Lazy<AtomicU64> = Lazy::new(|| AtomicU64::new(0));
+
+fn next_id() -> ProposeId {
+    ProposeId::new(NEXT_ID.fetch_add(1, Ordering::SeqCst).to_string())
```

Comment:
Why don't we use `Ordering::AcqRel` here?

Reply:
I agree. If the only purpose is to keep `ProposedId` unique, `AcqRel` is enough.

Reply:
Sure. Fixed